picked_count: 0
starting_vertx: 1
max_bins: 9
charge_destination: 7
